server/belga/search_providers.py

```diff
@@ -250,8 +250,8 @@ def fetch(self, guid):
         return self.format_list_item(data)
 
     def api_get(self, endpoint, params):
-        url = requests.Request('GET', 'http://example.com/' + endpoint, params=params).prepare().path_url
-        resp = self.session.get(self.url(url))
+        url = requests.Request('GET', self.url(endpoint), params=params).prepare().url
+        resp = self.session.get(url)
```

Comment:
so why not just do `resp = self.session.get(self.url(endpoint), params=params)` ?

Reply:
Hi, we tried to use like that. But when run unit-test, we don't know why if `expected params` and `actual params` are same. It raises this error.
`Expected call: mock('http://mules.staging.belga.be:48080/belga360-ws/archivenewsobjects/39670442')
Actual call: mock('http://mules.staging.belga.be:48080/belga360-ws/archivenewsobjects/39670442', params={})`
Property params is disappear in the `Expected call`.

If they are not same,
`Expected call: mock('http://mules.staging.belga.be:48080/belga360-ws/archivenewsobjects', params={'start': 50, 'pageSize': 50, 'searchText': 'test que'})
Actual call: mock('http://mules.staging.belga.be:48080/belga360-ws/archivenewsobjects', params={'start': 50, 'pageSize': 50, 'searchText': 'test query'})`
Property params is appear in the `Expected call`.

Maybe it is a bug in mock function.

Reply:
I see, can you add a comment about it?

Reply:
@idsvn , I'm not 100% sure if I got what is the issue with tests you have mentioned, but I think the test was wrong (now I see that it is fixed). Here https://github.com/superdesk/superdesk-belga/pull/75/commits/32bfe78cb032bb630937a06f0c7002942cc381f2#diff-68274e365c0d47a5bf2b1aff87837eb2L68 kwarg `params=params` was missed, that is why it was raising an error:
```
Expected call: mock('http://mules.staging.belga.be:48080/belga360-ws/archivenewsobjects/39670442')
Actual call: mock('http://mules.staging.belga.be:48080/belga360-ws/archivenewsobjects/39670442', params={})
```

Reply:
Hi Oleg, thank your support. It was fixed, because I miss add `params` in another unit-test function https://github.com/superdesk/superdesk-belga/pull/75/commits/2eab9ed280ac25e5d053ac4c190a304439390ad6 .